arc/conformance/plugin/arc_conformance.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
+
+set -x
+set -e
+
+results_dir="${RESULTS_DIR:-/tmp/results}"
+keyVaultName="arc-akv-secrets-$(openssl rand -hex 2)"
+keyvaultResourceGroup="akv-conformance-test-$(openssl rand -hex 2)"
+keyvaultLocation=westus2
+
+function waitForResources {
+    available=false
+    max_retries=60
+    sleep_seconds=10
+    RESOURCE=$1
+    NAMESPACE=$2
+    for i in $(seq 1 $max_retries)
+    do
+    if [[ ! $(kubectl wait --for=condition=available ${RESOURCE} --all --namespace ${NAMESPACE}) ]]; then
```

Comment:
The default timeout for `kubectl wait` is 30s. So for each loop this function will run 30s (wait) + 10s (sleep). We should instead just rely on the `kubectl wait` timeout and not have another sleep.